### ISSUE TITLE
Mac OS Version Instructions

### ISSUE THREAD

**Issue by jbreef** [input-leap/input-leap#2266]
### What happened?

The install for Mac OS requires two items to be added to the Accessibility section of Privacy & Security.  Both the InputLeap application and the input-leap exec file from the Contents/MacOS folder in the InputLeap.app folder need to be added to this section and enabled for InputLeap to execute properly.  Also, when upgrading to a new version the existing InputLeap items in Accessibility should be removed and replaced with the current version.

![Image](https://github.com/user-attachments/assets/be061f67-bff0-4b3f-90dc-24835d1786e1)

### Version

v3.0.3

### If applicable, where did you install Input Leap from?

Github

### What OSes are you seeing the problem on? (Check all that apply)

macOS

### If applicable, are you using Wayland or X11?

No

### What OS versions are you using?

15.5

### Relevant log output

```shell

```

**Comment by shymega** [comment]
This isn't a bug. Please open a PR to the wiki if you want this in the docs, but not an issue.
